app/bibleview-js/src/common.scss

```diff
@@ -160,3 +160,7 @@ $modal-content-background-color-night: black;
   width: 100%;
   overflow: hidden;
 }
+
+.small {
+    font-size: 0.9em
+}
```

Comment:
Why not in AmbiguousSelectionBookmarkButton.vue?

Reply:
better there as this definition is too generic ("small")

Reply:
here should be only those definitions that are used more widely in many components

Reply:
That's fine. But I would use 'small' as a generic way of getting smaller text. I didn't find an existing class that we could use for that. Does one exist?

Reply:
Why not putting it first in AmbiguousSelectionBookmarkButton and move it common when you find that you need that in multiple places? :-) Now it's only used in one place. But well, I'm not strict about this, if you want it to be in common, it's okay too.

Reply:
I also kind of don't want unexpected effects to happen all around the app, if I happen to want to adjust one font size. What works in one context as "smaller" font, might not work very well in other? That's why I like Vue's scoped styling.